go/ai/gen.go

```diff
@@ -271,7 +271,7 @@ type ModelResponseChunk struct {
 	Aggregated bool    `json:"aggregated,omitempty"`
 	Content    []*Part `json:"content,omitempty"`
 	Custom     any     `json:"custom,omitempty"`
-	Index      int     `json:"index,omitempty"`
+	Index      int     `json:"index"`
```

Comment:
Is this just so it's set to 0 if it's not set rather than be completely missing? Can't we set it in `GenerateWithRequest` if we see that it's missing?

Reply:
Yeah, I tried setting it explicitly to zero but the `omitempty` tag causes it to be ignored if it has its zero value.

Reply:
Ah true, good look.